setup.py

```diff
@@ -13,11 +13,20 @@
     sys.exit(1)
 
 
-if sys.version_info < (2, 6):
-    print("install failed - requires python v2.6 or greater", file=sys.stderr)
+if sys.version_info < (2, 7):
```

Comment:
While we are there, we should also check for python 3. And we should also set the `python_requires` argument in the `setup` function.

Reply:
A py3 port is coming soon (another dev is working on it) so I think I'll leave this for now.